### ISSUE TITLE
2.8.0 ignoring `root-modules`?

### ISSUE THREAD

**Issue by sellout** [ocharles/weeder#177]
I’m on Weeder 2.8.0 (#176, NixOS/nixpkgs@134eef34ebd558ac6808b6c3a8084317c1c20849) and it seems to be ignoring the `root-modules` setting.

I’m setting up Weeder on an existing project, so I started by adding everything that currently has issues to `root-modules`:
```toml
root-instances = [
    { class = '\.Bifoldable$' },
    { class = '\.Bifunctor$' },
    { class = '\.Bitraversable$' },
    { class = '\.Eq$' },
]
root-modules = [
    '^Unison\.Util\.BiMultimap$',
    '^Unison\.Util\.Logger$',
    '^Unison\.Util\.Relation$',
]
roots = [
    '^Main\.main$',
    '^Paths_.*',
]
type-class-roots = false
unused-types = true
```
The other fields all work as expected, and the syntax I used for `root-modules` matches [test/Spec/ModuleRoot.toml](https://github.com/ocharles/weeder/blob/bd6084776f85c668200f81d0236a59c7c7a459f4/test/Spec/ModuleRoot.toml), so I’m not sure what I’m doing wrong.

Also, I’m curious why the defaults add `^'Paths_.*'` to `roots` instead of to `root-modules`. I imagine the effect should be the same either way?

**Comment by ozkutuk** [comment]
`root-modules` were introduced by #157, which seems to be first released with 2.9.0. Interestingly enough, it is not mentioned in the changelog, but looking at the git tags that seems to be the case.

**Comment by sellout** [comment]
Thanks – I’m pretty sure I checked the changelog, but since I’m pretty bad at getting those right myself, I should have known better than to rely on that.
